scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/LightingModeListCell.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import javafx.scene.control.ListCell;
+
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+/**
+ * Converts the LightningMode values to strings to display in the open ComboBox.
+ */
+public class LightingModeListCell extends ListCell<LightingMode> {
+
+  @Override
+  protected void updateItem(LightingMode mode, boolean empty) {
+
+    super.updateItem(mode, empty);
+    if (mode == LightingMode.LIGHT) setText("Sun light only");
+    else if (mode == LightingMode.LIGHT_AND_SHADOWS) setText("Sun light with shadows");
+    else if (mode == LightingMode.NO_LIGHT) setText("No sun light effect");
+
+    else setText("Sun light only");
```

Comment:
A couple of things here. 
1. I remembered a sample discussion about the naming in the drop down boxes. I checked with Hudson and he said a decision had been made to go with the enum names for the drop boxes, but in plain english. See suggestion below.

2. With this code the way it is, it's wired up so that when the application is launched, it's setting the combobox to say "Sun light only" without listening in to what `LightingMode` the combobox is actually set to. For example, if for whatever reason the LightingMode came back as `null`, then the text would display "Sun light only" which wouldn't be accurate. 

To get over this, we can simply add a null check in both this class and ComboBoxStringConverter to be on the safe side, e.g.

 ```java
    super.updateItem(lightingMode, empty);
    if (lightingMode != null) {
      if (lightingMode == LightingMode.LIGHT) setText("Light");
      else if (lightingMode == LightingMode.LIGHT_AND_SHADOWS) setText("Light and shadows");
      else if (lightingMode == LightingMode.NO_LIGHT) setText("No light");
    }

```

In addition to this, in the controller, we can add in logic there to select the appropriate enum from the drop down (assuming we want the sample to launch on "no light"): see my suggestion there.